### ISSUE TITLE
Custom Temperature Overlay crash on loading save

### ISSUE THREAD

**Issue by stephen-dahl** [EtiamNullam/Etiam-ONI-Modpack#11]
I started getting this error today after using custom temperature overlay for weeks

```
[20:32:13.749] [1] [ERROR] World(Clone) Error in World(Clone).SimDebugView.OnPrefabInit
System.TypeInitializationException: The type initializer for 'CustomTemperatureOverlay.State' threw an exception. ---> System.MissingFieldException: Field 'SimDebugView/ColorThreshold.color' not found.
   --- End of inner exception stack trace ---
  at (wrapper dynamic-method) SimDebugView.OnPrefabInit_Patch1(object)
  at KMonoBehaviour.InitializeComponent () [0x00068] in <b5ce5ea9e2704a858d8e536cfe298967>:0
```

**Comment by EtiamNullam** [comment]
Looks like `Summer Update` broke this mod. I have disabled `CustomTemperatureOverlay` for everyone for now. I will push update soon.

Thanks for notice!

**Comment by stephen-dahl** [comment]
I'm working again after your last update, Thanks

**Comment by EtiamNullam** [comment]
Well, my previous update simply disabled the mod so it didn't crash anymore. Now I fixed it in 06fdeea.

Let me know if it works.

**Comment by stephen-dahl** [comment]
Ya I saw that it didn't crash and assumed it must be working 😄. I Saw this update come through and everything loads and looks good.
